tests/wpt/metadata/html/browsers/browsing-the-web/history-traversal/hashchange_event.html.ini

```diff
@@ -1,6 +1,4 @@
 [hashchange_event.html]
-  type: testharness
-  expected: TIMEOUT
   [Queue a task to fire hashchange event]
-    expected: TIMEOUT
+    expected: FAIL
```

Comment:
Why do we fail this test?

Reply:
Interestingly, Firefox passes both tests.

Reply:
Also it looks like bubbles should be false per the "Example" block at https://dom.spec.whatwg.org/#concept-event-fire.

Reply:
Given https://github.com/web-platform-tests/wpt/pull/11355 from 9 days ago, I think we should update 004.html to match.

Reply:
Ahh that explains it! I'll update it to `false`